app/js/SafeExec.js

```diff
@@ -87,3 +85,12 @@ function safeExecPromise(command, options) {
     })
   })
 }
+
+module.exports.runOnce = runOnce
+function runOnce(fn = function() {}) {
```

Comment:
Do we need the default implementation here? I feel like this function probably wouldn't be of much use without it.

Reply:
No, I'll remove it.